src/sql/sqlops.proposed.d.ts

```diff
@@ -894,6 +901,13 @@ declare module 'sqlops' {
 				 * undefined or the text is empty or undefined. The default level is error.
 				 */
 				message: DialogMessage
+
+				/**
+				 * Register an operation to run the back ground when the wizard is done
```

Comment:
Comment mentions wizards. Also i think this should probably be on the dialog itself rather than the tab